mapping/sustainability.yaml

```
@@ -783,8 +783,23 @@
     module: Climate finance
     indicator: Climate impact
     disclosure format: Disclose the cost per tonne of CO2 equivalent \[value, currency\].
-    mapping: Publish the cost in `environment.abatementCost`. If supporting documentation is available, publish in documents with `.documentType` set to 'abatementCostMethodology'.
-    example: ''
+    mapping: Publish the cost in `environment.abatementCost`. If supporting documentation is available, publish in `documents` with `.documentType` set to 'abatementCostMethodology'.
+    example: |-
+        {
+          "environment": {
+            "abatementCost": {
+              "amount": 50000,
```

Per example in the issue.

```suggestion
              "amount": 12.29,
```